src/atomate2/common/phonon_utils.py

```diff
@@ -0,0 +1,30 @@
+"""Define utilities for the phonon workflow."""
+from phonopy.units import VaspToTHz
+
```

Comment:
Can you move this code to `src/atomate2/common/jobs/phonons.py`. Seems like we don't need a whole file for this function.